runtime/src/main/java/org/corfudb/protocols/API.java

```diff
@@ -0,0 +1,1113 @@
+package org.corfudb.protocols;
```

Comment:
Can we please split this file as well? It's more than 1000 lines long. We can create a dedicated package for APIs, and have one file for each service.

Reply:
Definitely. That's next on my ToDo list.